tests/test_jobs.py

```diff
@@ -0,0 +1,64 @@
+from .utils import MockResponse, BaseTest
+from pyintelowl.main import cli
+from unittest import mock
+from requests import Session
+
+
+class TestJobs(BaseTest):
+    def test_job_help(self):
+        result = self.runner.invoke(cli, ["jobs", "-h"])
+        assert "Manage Jobs" in result.output
+        assert result.exception is None
+
+    @mock.patch.object(
```

Comment:
It would be better to create a wrapping decorator for this to choose to enable/disable this. Similar to how it's done with `mock_connections` decorator in IntelOwl's tests.